Resources/Prototypes/Entities/Mobs/NPCs/simplemob.yml

```diff
@@ -1,98 +1,22 @@
 - type: entity
   save: false
+  parent:
+  - BaseMob
+  - BaseMobDamageable
+  - BaseMobAtmosExposed
```

Comment:
Please god make the names for these shorter

Reply:
you clearly havent seen objective rework :trollface: this is very short

Reply:
Fixed in [`1798f23` (#19859)](https://github.com/space-wizards/space-station-14/pull/19859/commits/1798f23fdd477a21adb204479a638e0cb0d53d9d)